terraform.tf.j2

```diff
@@ -3,10 +3,10 @@ terraform {
 
   backend "s3" {
     bucket         = "{{terraform.state_file_bucket}}"
-    key            = "terraform/dataworks/aws-emr-template-repository.tfstate"
+    key            = "terraform/dataworks/aws-machine-learning-dev-cluster.tfstate"
```

Comment:
I don't think you want to change this in the template repository.

Reply:
oops

Reply:
done